library/src/main/java/com/android/volley/Request.java

```diff
@@ -145,6 +148,8 @@
     // request network time
     private long mRequestTime;
```

Comment:
This is the total time that the request took? Could we refactor the name to better differentiate from mRequestStartTime?